lib/sth_server.js

```diff
@@ -564,6 +646,70 @@ function startServer(host, port, aSTHDatabase, callback) {
   }
 
   /**
+   * Handler called once the notification info has been generated
+   * @param data Object including the following properties:
+   *  - contextElement The context element associated to the notification
+   *  - attribute The attribute associated to the notification
+   *  - recvTime The reception time associated to the notification
+   *  - counterObj The counter object
+   *  - totalTasks The total number of tasks to execute
+   *  - reply hapi's server reply() function
+   * @param err Error in case of error
+   * @param result The notification info in case no error ocurred
+   */
+  function onNotificationInfo(data, err, result) {
+    var request = data.request,
+        contextElement = data.contextElement,
+        attribute = data.attribute,
+        recvTime = data.recvTime,
+        counterObj = data.counterObj,
+        totalTasks = data.totalTasks,
+        reply = data.reply;
+
+    if (!err && !result.exists) {
+      // Store the raw data into the database
+      storeRawData(
+        {
+          request: request,
+          contextElement: contextElement,
+          attribute: attribute,
+          recvTime: recvTime,
+          notificationInfo: result
+        },
+        counterObj,
+        totalTasks,
+        reply
+      );
+
+      // Store the aggregated data into the database
+      storeAggregatedData(
+        {
```

Comment:
- Fixed in https://github.com/telefonicaid/fiware-sth-comet/commit/b08fbe8f87502e868ad09e72bffa4f31bb66b41a